base/traits.jl

```diff
@@ -11,6 +11,7 @@ OrderStyle(::Type{<:Real}) = Ordered()
 OrderStyle(::Type{<:AbstractString}) = Ordered()
 OrderStyle(::Type{Symbol}) = Ordered()
 OrderStyle(::Type{<:Any}) = Unordered()
+OrderStyle(::Type{Union{}}) = Ordered()
```

Comment:
Did you mean to use `Unordered()` as you wrote in the description?

It's hard to decide since `Union{}` is a true case of ambiguity which is included by multiple incompatible definitions from above. Maybe `Ordered()` is OK since `issorted(Union{}[])` is `true`.

Reply:
Woops I just saw you mentioned this above.